Jint.Benchmark/DromaeoBenchmark.cs

```diff
@@ -1,5 +1,5 @@
 ﻿using BenchmarkDotNet.Attributes;
-using Esprima.Ast;
+using Acornima.Ast;
```

Comment:
Just a though, what if there was a pre-PR that would introduce global usings for Esprima and it would then make this one smaller and maybe less against v3 development before merge?

Reply:
@adams85 I'll try out the PR for global using/type mapping and see how it would simplify current main